doc/quickstart.md

```diff
@@ -4,10 +4,18 @@
 
 1. A supported SRIOV hardware on the cluster nodes. Currently the supported models are:
   
-   | Vendor-ID   | Device-ID   |
-   | ----------- | ----------- |
-   | Intel       | 158b        |
-   | Melanox     | 1015, 1017  |
+| Model                    | Vendor ID | Device ID |
+| ------------------------ | --------- | --------- |
+| Intel XXV710 25GbE SFP28 |  0x8086 | 0x158b |
```

Comment:
hmm, just noticed this, for consistency either prefix the rest with 0x or remove the 0x from this entry whichever is clearer in your opinion.

Also appreciate if you squash the commit to one

Reply:
@ptrnull Since you have removed identifier like *GbE and SFP* from mellanox HW, I think its fine to remove from Intel HW too to make sure its consistent here. Thank you..

Reply:
Done @adrianchiris @martinkennelly